### PR TITLE
Edited README.md

### PR DESCRIPTION
**Pull request by matnguyen**
Edited README for concision and clarity.  Some sentences were rephrased to make them clearer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Winetricks
-Homepage of Winetricks, previously hosted at <https://code.google.com/p/winetricks>.
+The homepage for Winetricks was previously hosted at <https://code.google.com/p/winetricks>.
```
Comment (austin987) [inline README.md]: This is less clear IMO.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 Winetricks is an easy way to work around problems in Wine.
 
-It has a menu of supported games/apps for which it can do all the workarounds automatically. It also lets you install missing DLLs or tweak various Wine settings individually.
+It has a menu of supported games/apps for automatic workarounds. It also allows the installation of missing DLLs and tweaking of various Wine settings.
```
Comment (austin987) [inline README.md]: The first sentence is less clear to me. Second is arguably better.

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Tagged releases are accessible here:
 https://github.com/Winetricks/winetricks/releases
 
 # Installing
-If available and up to date, you should use the ```winetricks``` package, available in most distributions:
+The ```winetricks``` package should be used if it is available and updated. The package is available in most distributions:
```
Comment (austin987) [inline README.md]: "and updated" doesn't have the same meaning as "and up to date" in my opinion.

File: README.md [hunk 4]
```diff
@@ -21,15 +21,15 @@ If available and up to date, you should use the ```winetricks``` package, availa
 * Homebrew (OSX): http://brewformulas.org/Winetricks
 * MacPorts (OSX): https://www.macports.org/ports.php?by=name&substr=winetricks
 * Slackbuilds (Slackware): http://slackbuilds.org/repository/14.2/system/winetricks/?search=winetricks
-* Ubuntu: https://packages.ubuntu.com/search?keywords=winetricks Note: Ubuntu LTS versions are years out of date, do a manual install instead.
+* Ubuntu: https://packages.ubuntu.com/search?keywords=winetricks Note: Ubuntu LTS versions are years out of date, a manual installation should be done instead.
 
-If it's not available, outdated (e.g., Ubuntu LTSs), or you want the bleeding edge version, you can manually install winetricks:
+If the package is unavailable, outdated (e.g., Ubuntu LTSs), or newest version is wanted, a manual installation of winetricks can be done:
```
Comment (austin987) [inline README.md]: I'd prefer  "or the latest version is desired"

File: README.md [hunk 5]
```diff
@@ -38,13 +38,13 @@ sudo mv winetricks /usr/local/bin
 ```
 Note: /usr/local/bin must be in your $PATH for this to work.
 
-You can then keep winetricks up to date in the future by doing:
+Winetricks can be kept updated by doing:
```
Comment (austin987) [inline README.md]: "can be updated"

"kept" implies that it will be automatically updated in the future by that command, which isn't the case.

**Comment by matnguyen** [comment]
I fixed it up, hope it's clearer now :)

**Comment by matnguyen** [comment]
Sorry I really screwed this up.. first time squashing commits.  I'm gonna make a new pull request

**Pull request closed without merging**